client/executor/runtime-test/src/lib.rs

```diff
@@ -183,6 +183,12 @@ sp_core::wasm_export_functions! {
 		}
 	}
 
+
+	fn test_offchain_index_set() {
```

Comment:
what is the equivalent of this in a real runtime?

Reply:
`sp_io::offchain_index::set(b"k", b"v")`